app/models/user.ts

```diff
@@ -0,0 +1,12 @@
+import mongoose, { Schema } from "mongoose";
+
+const userSchema: Schema = new Schema({
+  _id: mongoose.Types.ObjectId,
```

Comment:
No need to have this as part of your model

Reply:
Removed _id from schema

Reply:
_id removed from schema